### PR TITLE
Sensors removed handler

### PR DESCRIPTION
**Pull request by nest1ing**
After resolves SRV-178 phosphor-hwmon send `InterfacesRemoved` signals.
This fix add handler for this signal.

### CODE REVIEW

**Review by artemsen — APPROVED** [review]

File: mibs/YADRO-MIB.txt [hunk 1]
```diff
@@ -126,7 +126,7 @@ yadroTempSensorName OBJECT-TYPE
     SYNTAX      OCTET STRING (SIZE(1..32))
     MAX-ACCESS  not-accessible
         -- smilint show warning if it have other access level
-        -- If set read-only it will be showed in snmptable output, 
+        -- If set read-only it will be showed in snmptable output,
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt]: "shown", not "showed"

File: mibs/YADRO-MIB.txt [hunk 2]
```diff
@@ -219,7 +219,7 @@ yadroVoltSensorName OBJECT-TYPE
...
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt]: "shown"

File: mibs/YADRO-MIB.txt [hunk 3]
```diff
@@ -312,7 +312,7 @@ yadroTachSensorName OBJECT-TYPE
...
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt]: shown

File: mibs/YADRO-MIB.txt [hunk 4]
```diff
@@ -405,7 +405,7 @@ yadroCurrSensorName OBJECT-TYPE
...
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt]: "shown"

File: mibs/YADRO-MIB.txt [hunk 5]
```diff
@@ -498,7 +498,7 @@ yadroPowerSensorName OBJECT-TYPE
...
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt]: "shown"

File: mibs/YADRO-MIB.txt [hunk 6]
```diff
@@ -595,7 +595,7 @@ yadroCurrSensorStateNotification NOTIFICATION-TYPE
         "Notification about changed state of current sensor"
     ::= { yadroNotifications 5 }
 
-yadroPowerSensorStateNotification NOTIFICATION-TYPE
+yadroPwrSensorStateNotification NOTIFICATION-TYPE
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt:598]: What's the purpose of this renaming?

File: mibs/YADRO-MIB.txt [hunk 7]
```diff
@@ -676,35 +676,35 @@ yadroTachSensorsTableGroup OBJECT-GROUP
...
-                  yadroTachSensorWarnHigh,
-                  yadroTachSensorCritLow,
-                  yadroTachSensorCritHigh,
-                  yadroTachSensorState
+    OBJECTS     { yadroCurrSensorValue,
+                  yadroCurrSensorWarnLow,
+                  yadroCurrSensorWarnHigh,
+                  yadroCurrSensorCritLow,
+                  yadroCurrSensorCritHigh,
+                  yadroCurrSensorState
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt:690]: This doesn't look related to the pull request summary.

File: mibs/YADRO-MIB.txt [hunk 8]
```diff
@@ -696,12 +696,12 @@ yadroCurrSensorsTableGroup OBJECT-GROUP
...
-                  yadroTachSensorWarnHigh,
-                  yadroTachSensorCritLow,
-                  yadroTachSensorCritHigh,
-                  yadroTachSensorState
+    OBJECTS     { yadroPowerSensorValue,
+                  yadroPowerSensorWarnLow,
+                  yadroPowerSensorWarnHigh,
+                  yadroPowerSensorCritLow,
+                  yadroPowerSensorCritHigh,
+                  yadroPowerSensorState
```
Comment (AlexanderAmelkin) [inline mibs/YADRO-MIB.txt:704]: Again, doesn't look related to the pull request summary

File: yadro/sdbus-helper.hpp [hunk 9]
```diff
@@ -0,0 +1,207 @@
+/**
+ * @brief sdbusplus helper
+ *
+ * This file is part of phosphor-snmp project.
+ *
+ * Copyright (c) 2018 YADRO (KNS Group LLC)
```
Comment (AlexanderAmelkin) [inline yadro/sdbus-helper.hpp]: Please use just `YADRO`

File: tests/emit-removed-ambient.sh [hunk 10]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+gdbus emit --system --object-path '/xyz/openbmc_project/sensors'            \
+           --signal org.freedesktop.DBus.ObjectManager.InterfacesRemoved    \
+           'objectpath "/xyz/openbmc_project/sensors/temperature/ambient"'  \
```
Comment (AlexanderAmelkin) [inline tests/emit-removed-ambient.sh:5]: Ehm... what's the use case? When can be an ambient temperature sensor removed?

File: mibs/YADRO-MIB.txt (hunk 6 above)
Comment (nest1ing) [reply mibs/YADRO-MIB.txt:598]: fix `smilint` warning:
```
mibs/YADRO-MIB.txt:598: object identifier name `yadroPowerSensorStateNotification' longer than 32 characters
```

File: tests/emit-removed-ambient.sh (hunk 10 above)
Comment (nest1ing) [reply tests/emit-removed-ambient.sh:5]: this command maybe used  in debug process to emulate of removing any sensors. 
`ambient` was chooses randomly.

**Review by AlexanderAmelkin — APPROVED** [review]
